script/deploy/safe/execute-pending-timelock-tx.ts

```diff
@@ -178,20 +178,60 @@ const cmd = defineCommand({
     if (isDryRun)
       consola.info('Running in DRY RUN mode - no transactions will be sent')
 
-    // Process each network
-    for (const network of networksToProcess)
-      try {
-        await processNetwork(
-          network,
-          isDryRun,
-          specificOperationId,
-          executeAll,
-          rejectAll,
-          rpcUrlOverride
-        )
-      } catch (error) {
-        consola.error(`Error processing network ${network.name}:`, error)
-      }
+    // Process networks - sequentially for interactive mode, parallel for auto-execute mode
+    if (executeAll || rejectAll) {
+      consola.info('🚀 Processing networks in parallel for auto-execution mode')
+
+      // Process all networks in parallel
+      const networkPromises = networksToProcess.map(async (network) => {
+        try {
+          return await processNetwork(
+            network,
+            isDryRun,
+            specificOperationId,
+            executeAll,
+            rejectAll,
+            rpcUrlOverride
+          )
+        } catch (error) {
+          consola.error(`Error processing network ${network.name}:`, error)
+          return { network: network.name, success: false, error }
+        }
+      })
+
+      // Wait for all networks to complete
+      const results = await Promise.all(networkPromises)
+
+      // Log summary
+      const successfulNetworks = results.filter(
+        (r) => r && r.success !== false
+      ).length
+      const failedNetworks = results.filter(
+        (r) => r && r.success === false
+      ).length
```

Comment:
These can be simplified as `processNetwork` will never return `null` or `undefined` because of its strict return type.

`(r) => r.success`
`(r) => !r.success`